api/src/main/java/org/openmrs/module/fhir/api/strategies/observation/ObservationStrategy.java

```diff
@@ -228,6 +227,11 @@ public Observation updateFHITObservation(Observation observation, String uuid) {
 		List<Observation> obsList = new ArrayList<>();
 		if (codings.isEmpty()) {
 			return obsList;
+		} else if (codings.size() == 1 && codings.get(0).getModifier() == null && codings.get(0).getSystem() == null) {
```

Comment:
Should we be supporting the case where the system is unset (null)?

Reply:
`if (codings.size() == 1 && codings.get(0).getModifier() == null && codings.get(0).getSystem() == null)` 
Looking for this case (TokenOrListParam - 5089AAAAAAAAAAAAAAAAAAAAA) where user has just pass concept uuid, both system and modifier are null